CODEOWNERS

```diff
@@ -81,6 +81,8 @@
 
 /Workbooks/Azure*SQL*VM/** @microsoft/azuresqlvm-workbook
 
+/Workbooks/Communication*Services/** @microsoft/acs-data
+
```

Comment:
you'll also want to add similar lines that cover your gallery files (see below where there are lines about gallery files)

Reply:
Thanks, just updated that to include the gallery files.